pkgs/os-specific/linux/kernel-headers/default.nix

```diff
@@ -49,5 +66,13 @@ in {
   linuxHeaders = common {
     version = "4.15";
     sha256 = "0sd7l9n9h7vf9c6gd6ciji28hawda60yj0llh17my06m0s4lf9js";
+    patches = [
+       ./no-relocs.patch # for building x86 kernel headers on non-ELF platforms
+       ./no-dynamic-cc-version-check.patch # so we can use `stdenvNoCC`, see `makeFlags` above
+       #(fetchpatch { # For building on non-ELF platforms
```

Comment:
If something is commented out, can you either include a bit of context on when it can/should be uncommented, or just delete it?